src/components/Filter.js

```diff
@@ -0,0 +1,75 @@
+// import * as json from "src/mock-data/tickets.json";
+import {useState} from "react";
+
+export function Filter() {
```

Comment:
Если в файле только один компонент реакта, используется дефолтгый экспорт
```suggestion
function Filter() {
...
}

export default Filter;